quickstarts/account-data-ingestion/config.yml

```
@@ -0,0 +1,37 @@
+# Name of the quickstart (required)
+name: acccount data ingestion
+
+# Displayed in the UI (required)
+title: Data Ingestion Breakdown
+
+# Long-form description of the quickstart (required)
+description: |
+  Shows bytes ingested for APM, Browser, Mobile, Infrastructure, Logs, Synthetics, and Kubernetes.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Shows bytes ingested for different entity types.
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+  - Sri Shetty
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - insights
+  - Data
+  - Ingest
```

I would remove insights here as it's a legacy product, there is a list of allowed keywords [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#keywords) but i think both `data` and `ingest` would be worth adding here
```suggestion
keywords:
  - data
  - ingest
```

---

Done